lib/src/features/splashScreen/view/splash_screen_view.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import 'package:myapp/src/router/coordinator.dart';
+import 'package:myapp/widgets/layout/base_layout.dart';
+
+class SplashScreenView extends StatelessWidget {
+  const SplashScreenView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BaseLayout(
+      buttonText: 'Get started',
+      onButtonPressed: () => AppCoordinator.showRegisterScreen(),
+      content: Column(
+        children: [
+          SvgPicture.asset('assets/svgs/splash_screen_bg.svg',
+              semanticsLabel: "Splash screen bg image", width: 200),
```

Comment:
what is this `semanticsLabel`?

Reply:
it the same with `alt` prop from image tag in html, i follow the guide from SvgPicture